### ISSUE TITLE
VcfBreakendToBedpe usage issues

### ISSUE THREAD

**Issue by cbrueffer** [PapenfussLab/gridss#249]
First of all, thanks for this software! I'm trying to use the `VcfBreakendToBedpe` functionality, but having some issues.

1. I'm trying to call the program as follows: `java -cp gridss.jar gridss.VcfBreakendToBedpe`, resulting in the following error:

```
# java -cp gridss.jar gridss.VcfBreakendToBedpe
Error: Could not find or load main class gridss.VcfBreakendToBedpe
Caused by: java.lang.ClassNotFoundException: gridss.VcfBreakendToBedpe
```

The above calling convention is what the gridss wrapper shipped with the gridss bioconda package uses internally, hence not using a `gridss.sh` example.

2. https://github.com/PapenfussLab/gridss#bedpe mentions a usage example in `examples/GRIDSS.sh`, however the file doesn't exist. `scripts/gridss.sh` does not use `VcfBreakendToBedpe`.

How should `VcfBreakendToBedpe` be executed?

**Comment by d-cameron** [comment]
>How should VcfBreakendToBedpe be executed?

It shouldn't  - apologies that the documentation was out of date. It has now been updated and an example script uploaded.

VcfBreakendToBedpe was removed because, from a workflow perspective, it didn't really make much sense. The default GRIDSS output is so verbose/sensitive that a workflow involving export to BEDPE would almost always include some sort of filtering step beforehand. Export to BEDPE loses so much information about the call that it should be performed as a final data format conversion step, not upfront directly from the GRIDSS output.

I wrote the `StructuralVariantAnnotation` R package to make the downstream analysis of SV calls (from any caller - not just GRIDSS) much simpler and easier to do. It makes a lot more sense to perform the export at the end of the filtering script instead of having to switch back and forward between R and java.
